app/scripts/modules/core/src/help/HelpMenu.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { Dropdown, MenuItem, Glyphicon } from 'react-bootstrap';
+
+import './HelpMenu.less';
+
+export const HelpMenu = () => {
+  return (
+    <li className="help-menu">
+      <Dropdown id="help-menu-dropdown" pullRight={true}>
+        <Dropdown.Toggle className="hidden-lg" noCaret={true}>
+          <Glyphicon glyph="question-sign" />
+        </Dropdown.Toggle>
+        {HelpMenuItems(true)}
+      </Dropdown>
+
+      <Dropdown id="help-menu-dropdown-large" pullRight={true}>
+        <Dropdown.Toggle className="hidden-xs hidden-sm hidden-md">
+          Help
+        </Dropdown.Toggle>
+        {HelpMenuItems()}
+      </Dropdown>
+    </li>
+  );
+};
+
+const HelpMenuItems = (divide?: boolean) => {
+  return (
+    <Dropdown.Menu>
+      <MenuItem href="https://spinnaker.io/docs" target="_blank">
+        Docs
+      </MenuItem>
+      {divide && <MenuItem divider={true} />}
+      <MenuItem href="https://spinnaker.io/community" target="_blank">
```

Comment:
Should this point to community.spinnaker.io instead?

Reply:
After a quick in-person with @lwander and @dorbin we're going with the link URL as it exists now but changing the content to "Community Resources".